### ISSUE TITLE
Duplicate records on stage site

### ISSUE THREAD

**Issue by gjost** [densho/encyc-rg#2]
The stage site (http://encycstage.densho.org/api/1.0/) shows lots of duplicate records.  These can be seen in the browse sections (e.g. http://encycstage.densho.org/api/1.0/browse/rg_rgmediatype/Books/) and also in the list pages (e.g. http://encycstage.densho.org/api/1.0/articles/).

**Comment by gjost** [comment]
I'm not seeing these with my local install which is running 4a204, same as the stage site.

**Comment by gjost** [comment]
I recall having these problems before with the namesdb.  IIRC it was a problem with Elasticsearch.  Something about reusing the same index (deleting and then recreating), or about aliases.

**Comment by gjost** [comment]
Deleted the current index (`encycfront-20170619a`) using the `encyc delete --index=encycfront-20170619a ==confirm` command from `encyc-core`.

Elasticsearch cat command no longer shows the index as present:
```
ansible@kingsize:/usr/local/src/encyc-rg$ curl 192.168.0.23:9200/_cat/indices?v
health status index                pri rep docs.count docs.deleted store.size pri.store.size 
green  open   encycfront-20170605    5   1       3015            2     36.8mb         18.5mb 
green  open   encycfront-20170308    5   1          1            0    607.5kb        303.7kb 
green  open   encycfront-20170602    5   1       3015            0     36.9mb         18.4mb 
green  open   ddrpublic-20170223-2   5   1     217041        14817    608.7mb        304.7mb 
green  open   ddrpublic-20170223     5   1     152635          742    385.4mb        192.7mb 
green  open   encycfront-20170310    5   1       3032           51     36.7mb         18.5mb 
green  open   namesdb-20170329a      5   1     301296            3    293.2mb        146.5mb 
```
Yet `encyc-rg` running with the following settings is still returning results.
```
DOCSTORE_HOSTS	[{'host': '192.168.0.23', 'port': '9200'}]
DOCSTORE_INDEX	'encycfront-20170619a'
```
There are no more duplicates but it should not be returning ANY results at this point.

**Comment by gjost** [comment]
Turns out that some of my indices (`encycfront-20170619a` and `encycfront-20170605` and `encycfront-20170602`) had gotten linked or aliased together.  I bet this happened as I did several rounds of `encyc create` and `encyc delete` using the same index name.  The problem was only solved when I deleted all of the offending indices.

This is pretty frustrating.  I don't know any database that has this problem.  There doesn't seem to be any way of detecting when indices are connected in this way.
